docs/build.gradle

```diff
@@ -27,9 +27,14 @@ task buildout (type:Exec, dependsOn: bootstrap) {
 }
 
 task itest (type:Exec) {
-  commandLine "$projectDir/bin/test", '-1'
+    commandLine "$projectDir/bin/test", '-1', '-t', '../(blob|sql|hello|best_practice)'
```

Comment:
is it possible to match everything except process_test here? Otherwise I suspect that we'll forget to add new tests here